docs/_i18n/en/documentation/table-options.md

```diff
@@ -524,6 +524,17 @@ The table options are defined in `jQuery.fn.bootstrapTable.defaults`.
         <td>false</td>
         <td>True to select checkbox or radiobox when clicking rows.</td>
     </tr>
+    <tr>
+        <td>ignoreClickToSelectOn</td>
+        <td>data-ignore-click-to-select-on</td>
+        <td>Function</td>
+        <td>{}</td>
```

Comment:
The default value is `<code>{ return $.inArray(element.tagName, ['A', 'BUTTON']); }</code>`

Reply:
Ahh, that's what the `{}` is, got it! I've updated the docs accordingly.